lib/test/fixtures/java_interop/System.java

```diff
@@ -0,0 +1,5 @@
+package java.lang;
```

Comment:
I didn't find a way to pass a code to `clj-kondo` as input for java class definition lint as we have for clojure, maybe this should be improved on clj-kondo side c/c @borkdude

Reply:
Can you explain in more words what you mean here?

Reply:
E.g. this works:

``` clojure
 $ cat /tmp/Foo.java
package dude;

public class Foo {

}

$ clj-kondo --lint /tmp/Foo.java --config '{:output {:analysis {:java-class-definitions true} :format :edn}}'
{:findings [], :summary {:error 0, :warning 0, :info 0, :type :summary, :duration 28, :files 0}, :analysis {:namespace-definitions [], :namespace-usages [], :var-definitions [], :var-usages [], :java-class-definitions [{:class "dude.Foo", :uri "file:/private/tmp/Foo.java", :filename "/private/tmp/Foo.java"}]}}
```

Reply:
@borkdude I mean using `-`, as the stdin, as we have for clojure code, otherwise we would always have to have the file on disk for tests

Reply:
OK, issue welcome